### DISCUSSION TITLE
How to enable `indent-blankline` scope lines

### DISCUSSION THREAD

**Discussion by vabatta** [catppuccin/nvim#638] [Q&A] [answered]
As per the images shown on https://github.com/lukas-reineke/indent-blankline.nvim#multiple-indent-colors and the integration within Catppuccin, how do I make the scope lines to appear?

In `catppuccin/nvim` loaded in `LazyVim`:
```lua
integrations = {
  indent_blankline = {
    enabled = true,
    colored_indent_levels = false,
  },
}
```

and in `lukas-reineke/indent-blankline.nvim`:
```lua
scope = {
  enabled = true,
  show_exact_scope = true,
},
```

**Comment by JustAnAverageGuy** [comment, marked as answer]
Hi, just figured it out. 
in `catpuccin/nvim`
```
integrations = {
    indent_blankline = {
       enabled = true,
       colored_indent_levels = true,
    },
}
```

and in `lukas-reineke/indent-blankline.nvim`

```
indent = {
    highlight = { 
         "RainbowRed",
          "RainbowYellow",
          "RainbowBlue",
          "RainbowOrange",
          "RainbowGreen",
          "RainbowViolet",
          "RainbowCyan",
    },
}
```

Essentially I guess `catpuccin/nvim` just defines those highlight groups' colors.

**Reply by MoltenMonster** [reply]
Thanks, the documentation doesn't make it too clear but this helped me as well.
